src/sonic-config-engine/sonic-cfggen

```diff
@@ -36,7 +36,7 @@ from portconfig import get_port_config, get_breakout_mode
 from sonic_py_common.multi_asic import get_asic_id_from_name, get_asic_device_id, is_multi_asic
 from sonic_py_common import device_info
 from swsscommon.swsscommon import ConfigDBConnector, SonicDBConfig, ConfigDBPipeConnector
-
+import asic_sensors_config
```

Comment:
Thanks @mlok-nokia for fixing vs issue, one minor comment- can we just import the get_asic_sensors_config() alone from asic_sensors_config

Reply:
> Thanks @mlok-nokia for fixing vs issue, one minor comment- can we just import the get_asic_sensors_config() alone from asic_sensors_config

Fixed as suggested.